src/test/resources/integration_tests.properties

```diff
@@ -2,3 +2,6 @@
 idam.api.url=http://localhost:${wiremock.server.port}/idam
 
 idam.s2s-auth.url=http://localhost:${wiremock.server.port}/s2s
+
```

Comment:
Is this file needed now that `src/test/java/uk/gov/hmcts/ccd/integrations/IntegrationTest.java` has been removed?

Reply:
Good point, will check and remove accordingly

Reply:
Fixed in b357d60